exonum/src/events/internal.rs

```diff
@@ -71,6 +82,41 @@ impl InternalPart {
                         }).map_err(|_| panic!("Can't execute shutdown"));
                         to_box(f)
                     }
+                    InternalRequest::VerifyTx(tx) => {
+                        let internal_tx = internal_tx.clone();
+                        let tx = Arc::new(tx);
```

Comment:
`tx` has to be cloned to be moved in a `FnMut` closure introduced below. I did not find another way to do that because `tx` is a trait object `Box<Transaction>` and has to remain so.